src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService userService = new UserService();
+        User user = userService.findByEmail(email);
+        if (user != null) {
+            return user.getEmail().equals(email) && user.getPassword().equals(password);
```

Comment:
I would say this check on email equality is not needed. We already know, that user here will be with correct email, because it was found by `userService.findByEmail` method.